packages/aws-cdk/README.md

```diff
@@ -430,7 +430,17 @@ For this reason, only use it for development purposes.
 **⚠ Note #2**: This command is considered experimental,
 and might have breaking changes in the future.
 
-**⚠ Note #3**: Expected defaults for certain parameters may be different with the hotswap parameter. For example, an ECS service's minimum healthy percentage will currently be set to 0. Please review the source accordingly if this occurs. 
+**⚠ Note #3**: Expected defaults for certain parameters may be different with the hotswap parameter. For example, an ECS service's minimum healthy percentage will currently be set to 0. Please review the source accordingly if this occurs.
+
+**⚠ Note #4**: Only usage of certain [CloudFormation intrinsic functions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference.html) are supported as part of a hotswapped deployment. At time of writing, these are:
+
+- `Ref`
+- `Fn::GetAtt`
```

Comment:
Maybe worth noting that `Fn::GetAtt` is only partially supported? It can only evaluate pre-defined attributes here:

https://github.com/aws/aws-cdk/blob/7df11f4d8475dbf6deefe8fa12df4fea0125b860/packages/aws-cdk/lib/api/evaluate-cloudformation-template.ts#L352-L365

Reply:
Thanks @tmokmss, will update. Is it reasonable to include a link to those predefined attributes in the code from this user facing doco as context? Or better to just put a general "only partially supported" qualifier against it?

Reply:
Thanks! Imo the latter is enough for README.md, since it's intended for end users after all.

Reply:
Have corrected